PatternPal/PatternPal.Extension/Views/StepByStepInstructionsView.xaml

```diff
@@ -145,6 +145,18 @@
                    MaxWidth="380"
                    Visibility="Hidden">
         </TextBlock>
+        <TextBlock x:Name ="incorrectTextBlock"
```

Comment:
Instead of making a new text block, I would change the value and color of the already existing text block

Reply:
Done.